src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
You should link to a CSS file, not a SCSS file. SCSS files are preprocessed to produce CSS files which browsers can understand.